### ISSUE TITLE
combo boxes or equivalent are inaccessible to those using the keyboard or screen readers

### ISSUE THREAD

**Issue by lilmike** [itchio/itch.io#1303]
I am a blind game developer, and there are a few spots on your website that I simply cannot use with my screen reader. Those using the keyboard would likely face similar issues. The biggest issue I find is that combo boxes, or their javascript powered equivalents, are not accessible. The combo boxes show up as an edit box to screen readers, but using the arrow keys or even typing into the box does not yield any useful feedback. I'd be willing to test any fixes, or if you'd like to test for yourself, I recommend voice over on mac (built in), NVDA on windows (free), and orca on linux (open source). Any other info you need let me know!
-Michael.

**Comment by leafo** [comment]
Hello,

Can you please specify which UI elements specifically you're having an issue with (the name of page, URL, or screenshot, will be helpful in identifying them). As far as I know the `<select>` html tag is accessible, and we use that where possible. I'm not sure what you mean when you refer to "combo box".

Thanks

**Comment by lilmike** [comment]
Hiya,
A rough list is below -- note that these are the only ones I've noticed:

1. The tag selection on list of games 
right after where you can sort by new, popular, new and popular, etc.
2. The genre selection on game page edits.
3. The tag selection on game page edits.
4. The collection select for adding a game to a collection I.E. which
collection to add the game to.

Again, there may be more, these are just the ones I've noticed. Given my
experience, it's likely anything using that particular widget will be
inaccessible.
-Michael.





On Mon, Mar 21, 2022 at 01:40:14PM -0700, leaf wrote:
>Hello,
>
>Can you please specify which UI elements specifically you're having an issue with (the name of page, URL, or screenshot, will be helpful in identifying them). As far as I know the `<select>` html tag is accessible, and we use that where possible. I'm not sure what you mean when you refer to "combo box".
>
>Thanks
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/itchio/itch.io/issues/1303#issuecomment-1074397050
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>

-- 
Sorry about the test, it's just a me thing.
Michael Taboada, Creator of Games. Does that make me a god, well, sorta.
My pgp key: 79BC390E8864CD71305D4DD606CD2197A6EE5ED7
My website: https://michaels.world
2MB website: https://2mb.games
Don't believe everything you read on the internet. I might not be real, you might not be real, and this email certainly isn't real.
Sent from my musically gifted...
stationary bicycle
